lib/modules/librarymodule/librarymodule.py

```diff
@@ -29,19 +30,145 @@ def interpret_message(self, msg: str, email, db, admin) -> Dict:
                     "command": Commands.LIBRARY_DONATE_BOOK,
                     "book_isbn": commands[1]
                 }
+            elif commands[0] == "borrow_book":
```

Comment:
tbh im not a massive fan of these massive if else chains. Python match statements/general dictionary stuff might be a better solution but not for now ig

Reply:
Yeah, I'll migrate to that when I have some free time next week.